openshift/oadp-operator
# OADP-4379: read ephemeral-storage in code

## Why the changes were made

Users were able to set `ephemeral-storage` values in DPA, but OADP code would never read the values. This PR fixes this behavior.

## How to test the changes made

Create OADP with `make deploy-olm` and create DPA with `ephemeral-storage` values. Check if values are passed to Velero and node-agent containers.
